Lib/test/test_readline.py

```diff
@@ -9,14 +9,21 @@
 import sys
 import tempfile
 import unittest
-from test.support import import_module, unlink, temp_dir, TESTFN
+from test.support import import_module, unlink, temp_dir, TESTFN, verbose
 from test.support.script_helper import assert_python_ok
 
 # Skip tests if there is no readline module
 readline = import_module('readline')
 
 is_editline = readline.__doc__ and "libedit" in readline.__doc__
 
+
+def setUpModule():
+    if verbose and hasattr(readline, "_READLINE_VERSION"):
+        print(f"readline version: {readline._READLINE_VERSION:#x}")
+        print(f"readline runtime version: {readline._READLINE_RUNTIME_VERSION:#x}")
```

Comment:
We add these with:

    PyModule_AddIntConstant(m, "_READLINE_VERSION", RL_READLINE_VERSION);

So maybe we don't need the hasattr check?

Also, we need to know this is libedit, and looking at the tests, we need:

    is_editline = readline.__doc__ and "libedit" in readline.__doc__